src/constants/screens.ts

```diff
@@ -1 +1,5 @@
+// Auth
 export const WELCOME_SCREEN = 'WelcomeScreen';
+
+// Main
+export const HOME_SCREEN = 'HomeScreen';
```

Comment:
these variables are necessary? you are already using "enum AppScreens....".

Reply:
totally!